core-java-modules/core-java-streams-simple/src/test/java/com/baeldung/streams/minmaxbygroup/OrderProcessorUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.streams.minmaxbygroup;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class OrderProcessorUnitTest {
+  @Test
+  void whenOrderItemsAreGrouped_thenGetsMinMaxPerGroup() {
+    List<OrderItem> items =
+        Arrays.asList(
+            new OrderItem(1L, OrderItemCategory.ELECTRONICS, 1299.99),
+            new OrderItem(2L, OrderItemCategory.ELECTRONICS, 1199.99),
+            new OrderItem(3L, OrderItemCategory.ELECTRONICS, 2199.99),
+            new OrderItem(4L, OrderItemCategory.FURNITURE, 220.00),
+            new OrderItem(4L, OrderItemCategory.FURNITURE, 200.20),
+            new OrderItem(5L, OrderItemCategory.FURNITURE, 215.00),
+            new OrderItem(6L, OrderItemCategory.CLOTHING, 50.75),
+            new OrderItem(7L, OrderItemCategory.CLOTHING, 75.00),
+            new OrderItem(8L, OrderItemCategory.CLOTHING, 75.00));
+
+    OrderProcessor orderProcessor = new OrderProcessor();
+    final Map<OrderItemCategory, Pair<Double, Double>> orderItemCategoryPairMap =
+        orderProcessor.groupByCategoryWithMinMax(items);
+    Assertions.assertEquals(
```

Comment:
you can import static the Assertions and leave each assertEquals in one single line

Reply:
Done. I had to format that manually. I was using intellij-baeldung-style.xml, then the formatting takes it to next line.